README.md

```diff
@@ -49,6 +49,11 @@ This is a list of some wonderful open-source projects & applications integrated
 * [lightseq](https://github.com/bytedance/lightseq) (from ByteDance) - A high performance inference library for sequence processing and generation implemented in CUDA.
 * [FastSeq](https://github.com/microsoft/fastseq) (from Microsoft) - Efficient implementation of popular sequence models (e.g., Bart, ProphetNet) for text generation, summarization, translation tasks etc.
 
+## 🌗 Model Parallelism
```

Comment:
```suggestion
## 🌗 Model Scalability
```

Let's make a generic section on scalability, where parallelism is one of the ways.

Reply:
Cool !